lms/static/js/student_account/components/StudentAccountDeletionModal.jsx

```diff
@@ -168,7 +162,9 @@ class StudentAccountDeletionConfirmationModal extends React.Component {
                                                 <span>{bodyDeletion2}</span>
                                             </p>
                                             {/* eslint-disable-next-line react/no-danger */}
-                                            <p dangerouslySetInnerHTML={{ __html: loseAccessText }} />
+                                            <p>
+                                                <span>{loseAccessText}</span>
+                                            </p>
```

Comment:
Sorry this came in as 2 reviews instead of one!

```suggestion
                                            <p>{loseAccessText}</p>
```

Reply:
Sorry! I missed that one too. Thank you.